src/pocketmine/network/mcpe/protocol/types/ResourcePackType.php

```diff
@@ -30,9 +30,13 @@ private function __construct(){
 	}
 
 	public const INVALID = 0;
-	public const RESOURCES = 1;
-	public const BEHAVIORS = 2;
-	public const WORLD_TEMPLATE = 3;
-	public const ADDON = 4; //scripts?
-	public const SKINS = 5;
+	public const ADDON = 1;
+	public const CACHED = 2;
+	public const COPY_PROTECTED = 3;
+	public const BEHAVIOR = 4;
+	public const PERSONA_PIECE = 5;
+	public const RESOURCES = 6;
+	public const SKINS = 7;
+	public const WORLD_TEMPLATE = 8;
+	public const COUNT = 9;
```

Comment:
Cargo-cult code.

It's common practice in C++ to add a dummy member to the end of an enum for range-checking, like so:

```
enum things: uint8_t {
    thing1,
    thing2,
    thing3,
    number_of_things
}
```
Here, this doesn't make any sense, because this is not an enum.